test/test-xml.js

```diff
@@ -0,0 +1,279 @@
+/* eslint-disable no-undef */
```

Comment:
Hi @joshuaeke , this is a public repo - we must not commit client MARC data to this

Reply:
Apologies, didn't realise this was a public repository, removing the data from the commit history now.

Reply:
HI @danieljtribe, 

I'm curious about the request to remove the test file I added. I've already taken out any 'Public MARC information from the client,' as I agree that's a security concern.

The 'test-xml.js' file now includes a MARC record with randomized fields that aren't tied to any specific client. Should we keep this test file in the repository, or are we aiming to remove all MARC data? If so, could you share the reasoning behind it?